File: Java/Produk.java
public class Produk {
    private int id;
    private String judul;
    private int durasi;

    public Produk(int id, String judul, int durasi){
        this.id = id;
        this.judul = judul;
        this.durasi = durasi;
    }
    public int getId(){ return id; }
    public String getJudul(){ return judul; }
    public int getDurasi(){ return durasi; }
    public void setJudul(String judul){ this.judul = judul; }
    public void setDurasi(int durasi){ this.durasi = durasi; }
}

File: Python/produk.py
class Produk:
    def __init__(self, id, judul, durasi):
        self.id = id
        self.judul = judul
        self.durasi = durasi

    def get_id(self):
        return self.id

    def get_judul(self):
        return self.judul

    def get_durasi(self):
        return self.durasi

    def set_judul(self, judul):
        self.judul = judul

    def set_durasi(self, durasi):
        self.durasi = durasi